docs/maps/maps-aggregations.asciidoc

```diff
@@ -68,9 +68,9 @@ To enable a blended layer that dynamically shows clusters or documents:
 
 [role="xpack"]
 [[maps-top-hits-aggregation]]
-=== Top hits per entity
+=== Display the most relevant entities
```

Comment:
Looks like your changes from your other PR made it into this PR. How about reverting changes to this file since there is an open PR to make these changes?